cmake/Install.cmake

```diff
@@ -633,7 +641,7 @@ cpack_ifw_configure_component(Unspecified
 )
 
 cpack_ifw_configure_component(Symlinks
-    SCRIPT cmake/qtifw/install_mac_createsymlinks.qs
+    SCRIPT cmake/qtifw/install_unix_createsymlinks.qs
```

Comment:
I see, you just renamed it from Mac to Unix to make it general for both cases.  OK.